maths/find_max_recursion.py

```diff
@@ -0,0 +1,21 @@
+# Divide and Conquer algorithm
+def find_max(nums, left, right):
+    """
+    find max value in list
+    :param nums: contains elements
+    :param left: index of first element
+    :param right: index of last element
+    :return: min in nums
```

Comment:
:return: says min but should say max.

Also, add doctests as discussed in CONTRIBUTING.md
```python
>>> nums = [1, 3, 5, 7, 9, 2, 4, 6, 8, 10]
>>> find_max(nums, 0, len(nums) - 1) = max(nums)
True
```

Reply:
Updated, Thanks for your review! @cclauss

Reply:
Both files need doctests like the one above.

Reply:
Please review again!

Reply:
> :return: says min but should say max.
> 
> Also, add doctests as discussed in CONTRIBUTING.md
> 
> ```python
> >>> nums = [1, 3, 5, 7, 9, 2, 4, 6, 8, 10]
> >>> find_max(nums, 0, len(nums) - 1) = max(nums)
> True
> ```

I update as you say, correct?